Library/Extensions/NfieldConnectionClientExtensions.cs

```diff
@@ -29,12 +30,10 @@ namespace Nfield.Extensions
     /// </summary>
     public static class NfieldConnectionClientExtensions
     {
-        private static string BackgroundActivityUrl(this INfieldConnectionClient client, string activityId)
+        private static Uri BackgroundActivityUrl(this INfieldConnectionClient client, string activityId)
         {
-            var result = new StringBuilder(client.NfieldServerUri.AbsoluteUri);
-            result.AppendFormat(CultureInfo.InvariantCulture, @"BackgroundActivities/{0}", activityId);
-
-            return result.ToString();
+            return new Uri(client.NfieldServerUri, string.Format(CultureInfo.InvariantCulture,
```

Comment:
Can we start using string interpolation when we refactor code?

Reply:
not in the general case, unless we add a nuget package to make those invariant culture.